pkg/custompluginmonitor/plugin/plugin.go

```diff
@@ -55,41 +55,53 @@ func (p *Plugin) GetResultChan() <-chan cpmtypes.Result {
 func (p *Plugin) Run() {
 	runTicker := time.NewTicker(*p.config.PluginGlobalConfig.InvokeInterval)
```

Comment:
I think it's good to also clean up this code a little bit.
We could add a `defer runTicker.Stop()` line after the initialization of runTicker:
`runTicker := time.NewTicker(*p.config.PluginGlobalConfig.InvokeInterval)`

Reply:
Good catch.